Riot/Categories/UIColor.swift

```diff
@@ -17,11 +17,11 @@
 import UIKit
 
 @objc extension UIColor {
-    convenience init(rgb: UInt) {
+    convenience init(rgbHex: UInt) {
```

Comment:
Note that the Material cocoapod which was imported to handle some UI presentation tasks also provides a (conflicting and incompatible) UIColor.init(rgb) convenience method.  We needed to update our method to avoid conflicts.